netlify/build

```
@@ -23,10 +23,12 @@ cp _site/docs/404.html _site/404.html
 
 # Set up htmltest
 
-curl https://htmltest.wjdp.uk | bash
-echo "debug 0"
+curl -s https://htmltest.wjdp.uk | bash
+if [[ $? != 0 ]]; then
+	echo "Failed to install htmltest"
+	exit 1
+fi;
```

```suggestion
fi
```
It will work but it's non-standard syntax.

---

There are other parts of this file that have it, so I'll log a new PR for it on Monday.